ffw/UIRPC.js

```diff
@@ -287,6 +287,33 @@ FFW.UI = FFW.RPCObserver.create(
             }
             break;
           }
+          case 'UI.CancelInteraction':
+          {
+            var targetID = request.params.cancelID;
+            var typeID = request.params.functionID;
+
+            if (typeID === 10 && SDL.InteractionChoicesView.active
+               && (targetID === undefined || targetID === SDL.InteractionChoicesView.cancelID)) {
+              SDL.InteractionChoicesView.deactivate('ABORTED');
+            } else if (typeID === 12 && SDL.AlertPopUp.active
+               && (targetID === undefined || targetID === SDL.AlertPopUp.cancelID)) {
+              SDL.AlertPopUp.deactivate();
+            } else if (typeID === 25 && SDL.ScrollableMessage.active
+               && (targetID === undefined || targetID === SDL.ScrollableMessage.cancelID)) {
+              SDL.ScrollableMessage.deactivate(true);
+            } else if (typeID === 26 && SDL.SliderView.active
+               && (targetID === undefined || targetID === SDL.SliderView.cancelID)) {
+              SDL.SliderView.deactivate(false);
```

Comment:
```suggestion
              SDL.SliderView.deactivate();
```
Apparently false indicates a `SUCCESS` result, no idea why. You apparently need to omit to parameter to send `ABORTED`.